### ISSUE TITLE
Using Switch.Thumb animation prop throws hydration error

### ISSUE THREAD

**Issue by johnsimeroth** [tamagui/tamagui#1584]
### Current Behavior

Using animations for switch with NextJS project throws a hydration error. Error details below. Animation still works after closing out the error message. No issues with hot refresh, just when I click the refresh button on chrome or open for first time.

Unhandled Runtime Error
Error: Hydration failed because the initial UI does not match what was rendered on the server.
Warning: Expected server HTML to contain a matching <span> in <span>.
See more info here: https://nextjs.org/docs/messages/react-hydration-error
Component Stack
span
span
eval
SwitchThumb2
button
span
span
eval
Provider
... etc ....

### Expected Behavior

No hydration errors.

### Tamagui Version

```markdown
latest (1.57.1)
```

### Reproduction

```markdown
Needed NextJS so instead of forking one of the aforementioned repos, I reproduced by:
1. run npm create tamagui@latest
2. Navigate to myapp/packages/app/features/home/screen.tsx
3. Replace the entire contents of the file with:
import { YStack, Switch } from '@my/ui'
export function HomeScreen() {
  return (
    <YStack f={1} jc="center" ai="center" p="$4" space>
      <Switch>
        <Switch.Thumb animation='quick' />
      </Switch>
    </YStack>
  )
}
4. Run yarn web
5. open chrome to localhost:3000
This should throw the errors. If not, refresh chrome once.
```

### System Info

```markdown
System:
    OS: macOS 13.5
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
    Memory: 614.10 MB / 32.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 16.19.0 - ~/.nvm/versions/node/v16.19.0/bin/node
    Yarn: 3.5.0 - ~/.nvm/versions/node/v16.19.0/bin/yarn
    npm: 9.6.4 - ~/.nvm/versions/node/v16.19.0/bin/npm
  Browsers:
    Chrome: 116.0.5845.110
    Safari: 16.6
  npmPackages:
    @babel/runtime: ^7.18.9 => 7.21.0 
    @manypkg/cli: ^0.19.1 => 0.19.2 
    check-dependency-version-consistency: ^3.0.3 => 3.0.3 
    eslint: ^8.21.0 => 8.39.0 
    node-gyp: ^9.3.1 => 9.3.1 
    prettier: ^2.7.1 => 2.8.8 
    turbo: ^1.10.3 => 1.10.3 
    typescript: ^5.1.3 => 5.1.3
```

**Comment by nkomissar** [comment]
I have the same problem with animation prop on Accordion.

**Comment by johnsimeroth** [comment]
I'm getting the same error on every component I've tried using it with so far. However, I found that when I switch out the animation driver to `@tamagui/animations-css` this problem goes away on web. So it seems to be specific to RN animations on web?

My assumption was that because @tamagui/animations-react-native includes drivers for RN and RNW, I could still use it. And it did seem to produce the animations I wanted even on web, just throws the hydration error whenever you do a full refresh. Would prefer to keep the RN option for web too if possible, since CSS only option doesn't support simple springy animations.

**Comment by natew** [comment]
this is for moti or react-native animation driver? i'll have to revisit the rn one, moti should work as well.

**Comment by johnsimeroth** [comment]
This is with @tamagui/animations-react-native, which is the driver that comes pre installed when you run `npm create tamagui`. I didn't see moti listed here (https://tamagui.dev/docs/core/configuration#animations) but I can try that one.

**Comment by natew** [comment]
fix in 1.63 going out now
